tools/codegen/cmd/mixgenproc/BUILD

```diff
@@ -0,0 +1,14 @@
+package(default_visibility = ["//visibility:public"])
```

Comment:
I agree, makes more sense in top level `cmd` dir. I think it'll help discoverability long term.